src/content/defi/index.md

```diff
@@ -0,0 +1,177 @@
+---
+title: Decentralized finance (defi)
+description: An overview of defi on Ethereum
+lang: en
+template: use-cases
+sidebar: true
+image: ../../assets/use-cases/defi.png
+sidebarDepth: 2
+summaryPoints:
+  [
+    "A global, open alternative to the current financial system.",
+    "Probably the biggest use case for Ethereum right now.",
+    "Powered by smart contracts on the Ethereum blockchain.",
+  ]
+---
+
+Defi is probably the biggest Ethereum success story so far.
+
+## What's defi?
+
+Defi is a collective term for products that are making financial products and services accessible to anyone who can use Ethereum – anyone with an internet connection. With defi, there are no centralized authorities who can block payments or deny you access to anything.
+
+There's a booming crypto economy out there, where you can lend, borrow, long/short, earn interest, and more. Crypto-savvy Argentinians have used defi to escape crippling inflation. Companies have started streaming their employees their wages in real time. Some folks have even taken out and paid off loans worth millions of dollars without the need for any personal identification.
+
+Right now there's ~ 40 billion dollars worth of crypto in defi applications. This is defi.
+
+| Defi                        | TradFi                                                   |
+| --------------------------- | -------------------------------------------------------- |
+| You control your funds      | Your funds are looked after by companies                 |
```

Comment:
1) I wonder if control & custody are worth differentiating 🤔 
2) There's probably a better term than "looked after"... "held by", "controlled by"? In essence we put trust in companies to be responsible while holding our funds & e.g. not to lend them out to risky borrowers. Not sure if it's worth explaining anywhere what actually happens to your bank deposits or not & the concept of fractional reserves.